src/client.js

```diff
@@ -114,6 +114,14 @@ export function commitContainer(system, commitData) {
     });
 }
 
+export function runHealthcheck(system, id) {
+    return new Promise((resolve, reject) => {
+        podmanCall("libpod/containers/" + id + "/healthcheck", "GET", {}, system)
```

Comment:
This is already a promise so you can return it directly.

Reply:
Updated